contracts/ProofOfExistence2.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.4.4;
+
+contract ProofOfExistence2 {
+    bytes32[] private proofs;
+
+    function storeProof(bytes32 proof) {
+        proofs.push(proof);
```

Comment:
hm, I don't know why that is. It seemed straightforward with an array of byte32.